lambdas/services/dynamo_service.py

```diff
@@ -10,32 +10,36 @@
 
 
 class DynamoDBService:
-    def __init__(self, table_name):
+
+    def get_table(self, table_name):

```

Comment:
If we're removing the init method, then there will be no point in this being a class anymore. We should de-class this and just have methods to do the needful, I think. The only references to self are self.get_table, and that in itself is a static method.